sql/moz-fx-data-shared-prod/subscription_platform_derived/google_subscriptions_v1/metadata.yaml

```diff
@@ -0,0 +1,12 @@
+friendly_name: Google Subscriptions
+description: >
+  Google subscriptions.
+owners:
+  - srose@mozilla.com
+labels:
+  schedule: daily
```

Comment:
should we include `incremental: false` label here? Or at least that's what the query looks like?

Reply:
Sure, I don't mind adding the `incremental` label to clarify that.

It seems like we aren't very consistent overall about specifying `incremental` labels though:  of 742 `metadata.yaml` files in `sql/` only 237 have an `incremental` label (182 true, 55 false).